creature/themanclaw; Drow Expansion Pack.json

```diff
@@ -0,0 +1,2113 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "DROWS",
+				"abbreviation": "DROWS",
+				"full": "Drow Expansion Pack(version 1.0)",
+				"authors": [
+					"themanclaw"
+				],
+				"convertedBy": [
+					"p4535992"
+				],
+				"version": "2020.03.21",
+				"url": "https://imgur.com/a/P8S84",
+				"targetSchema": "1.6.3"
+			}
+		],
+		"dateAdded": 1555319662,
+		"dateLastModified": 1555319662
+	},
+	"monster": [
+		{
+			"name": "DROW - Drow Stalker",
+			"size": "M",
+			"type": {
+				"type": "humanoid",
+				"tags": [
+					"Elf",
+					"Drow",
+					"Stalker"
+				]
+			},
+			"source": "DROWS",
+			"alignment": [
+				"N",
+				"E"
+			],
+			"ac": [
+				{
+					"ac": 16,
+					"from": [
+						"studded leather"
+					]
+				}
+			],
+			"hp": {
+				"formula": "11d8 + 11",
+				"average": 60
+			},
+			"speed": {
+				"walk": 30
+			},
+			"str": 12,
+			"dex": 18,
+			"con": 12,
+			"int": 13,
+			"wis": 15,
+			"cha": 12,
+			"passive": 17,
+			"cr": "6",
+			"shortName": "Drow Stalker",
+			"fluff": {
+				"entries": [
+					"<div class=\"note-editable\" style=\"max-height: 300px;\" contenteditable=\"true\"><p><a href=\"https:\/\/imgur.com\/a\/P8S84\">Drow Stalker<\/a><\/p><p><span class=\"userscript-fontstyle0\">Drow Stalkers are expert hunters of sentient prey. They attack unseen, and by the time their enemies know what is happening, it is already too late, as they are hit with a barrage of poisonous arrows and flurry of blades from shadows in all directions. In the rare event that an enemy manages to flee, they are quickly overtaken by the skilled trackers, only to realize their fate is inescapable.<\/span> \r\n<br style=\"font-style: normal ; font-variant: normal ; font-weight: normal ; letter-spacing: normal ; line-height: normal ; text-indent: 0px ; text-transform: none ; white-space: normal ; word-spacing: 0px\"><\/p><\/div>"
```

Comment:
While `"entries"` will accept arbitrary HTML, it's more "bug" than feature. This is borderline abusive. I don't want to write a HTML sanitiser and add whitelisting for the brews that actually need it (with big scary warnings attached), but...